contrib/node/src/python/pants/contrib/node/register.py

```diff
@@ -37,6 +38,7 @@ def register_goals():
   task(name='node', action=NodeResolve).install('resolve')
   task(name='node', action=NodeRun).install('run')
   task(name='node', action=NodeTestTask).install('test')
+  task(name='node', action=NodeBundle).install('bundle')
 
```

Comment:
To register your new target type, you'd need to include it in the dict of `targets={..}` above.

The fact that it isn't actually registered and your integration tests are passing would seem to indicate that usage of the target is optional. Is that intentional?